index.android.js

```diff
@@ -0,0 +1,12 @@
+import Scenario from './scenario'
+
+const thrower = async () => {
+  throw new Error('not supported on Android yet!')
+}
+
+export default {
+  initialize: thrower,
+  scan: thrower,
+  prepareDatabase: thrower,
+  Scenario,
```

Comment:
Why pass scenario at all? Just wondering

Reply:
idk, it's a constant unlike the other functions, so i was just trying to avoid a `can't read X of undefined` error